script/GameObjects/Entities/Ship/ShipInvis.lua

```diff
@@ -19,15 +19,15 @@ local Background = subclass(Entity, function (self, proto)
   self:setDrag(0.75, 4.0)
   self:setScale(proto.scale)
 
-  local mass = 50.0 * (self:getRadius() ^ 3.0)
```

Comment:
Not sure what this was before. volume (??mass??) of sphere is 4/3 * pi * r^3, but 4/3*pi is more like 4 than 50.
I guess it depends what shape is intended also.
In any case I assume that the fixed value is temporary, but do wee need to keep the original formula as a comment in case?

Reply:
This "40000" mass value applies only to the "invisible ship," which is used only in Startup Mode to show a slowly-rotating background while the Main Menu (or splash screen) is displayed. It's actually the ship that's turning, not the star system; and the mass value determines the ship's rotation rate. 40000 is a value observed after considerable testing to provide a nice, slow-but-not-too-slow rotation rate. The original calculation is no longer used anywhere, including the main game (it has been replaced by a different calculation of a ship's mass). I have updated ShipInvis.lua to clarify how this mass value is used.